### DISCUSSION TITLE
[Pesquisa Salarial] Quais os salários de Desenvolvedores pelo Brasil?

### DISCUSSION THREAD

**Discussion by penelopevitoria** [frontendbr/forum#1447] [Discussão]
### E aí, pessoal. Tudo certinho? 

Estou aqui hoje representando a **GeekHunter**, que é uma plataforma de _matching_ entre _DEVs_ e empresas de tecnologia. Nesses 3 anos de atuação já acumulamos milhares de parceiros em nossa comunidade. Sim, agora somos uma comunidade de quase **100k de pessoas** :)

Desde o início da nossa trajetória nós nos deparamos com uma grande dor, tanto dos desenvolvedores, como das empresas. Da parte dos profissionais existia **uma grande dúvida quanto ao seu valor de mercado em contextos diferentes**, como cidade, tecnologia principal, entre outros pontos.

É por isso que nós estamos fazendo uma pesquisa de salários para ajudar desenvolvedores de regiões, focos de carreira e tecnologias diferentes a entenderem melhor o seu valor de mercado. Logo, nada melhor do que pedir a ajuda dos nossos principais parceiros nesse mercado: **vocês**. 

Em resumo, nós queremos saber quais os salários de Desenvolvedores de todo o Brasil. Com essas informações criaremos um material que será compartilhado com todos os participantes, quando pronto. No caso, postaremos ele aqui :)

O questionário é bem curtinho e vocês não levarão mais do que **5 minutinhos** respondendo. E aí, topam? :)

### **O link está aqui:** http://bit.ly/31i8yqY

Desde já agradecemos muito. Um abraço!

**Comment by willycamargo** [comment]
Interessante iniciativa, mas na minha opinião a pesquisa esta bem rasa e mal formulada.
Várias questões são importantes para serem cruzadas e entenderem o salário final de um colaborador.

Não abrange trabalho remoto por exemplo, o que pode mascarar bem salários de quem mora no interior e trabalha remoto.

Também não pergunta sobre tempo de experiência, formação, responsabilidades, carga horária etc.

Acho que vale baixar a pesquisa do Stack Overflow (que também tem algumas falhas) e fazer um filtro para os resultados do  BR:
https://insights.stackoverflow.com/survey

**Comment by felipefialho** [comment]
Soltamos um ano passado se interessar -> https://survey.frontendbr.com.br/
